src/doctor/cli.py

```diff
@@ -42,10 +42,22 @@ def parse_arguments():
     parser.add_argument("--verbose", "-v", dest="log_level",
                         action="count", default=2,
                         help='Verbose logging. More options give higher verbosity.')
+    parser.add_argument("--list", nargs='?', const='*', metavar="PATTERN",
+                        help=("List rules matching pattern. "
+                              "If no pattern is given, will list all rules"))
+    parser.add_argument(
```

Comment:
Looking at the code, and the usage, I didn't understand that `--show` should be used together with `--list`. I think that the program should abort if `--show` is used without `--list`.

Reply:
That is correct. Fixed that and here is the result:

```
mats@fury:~/work/timescale/doctor$ timescale-doctor --show=brief
usage: timescale-doctor [-U USER] [-d DBNAME] [-p PORT] [-h HOSTNAME] [--help] [--verbose] [--list [PATTERN]] [--show {brief,message,details}] [DBNAME]
timescale-doctor: error: called with '--show' but without '--list'
mats@fury:~/work/timescale/doctor$ timescale-doctor --list --show=brief
index.UnusedIndex:
    Find all unused indexes.
hypertable.HypertableCandidate:
    Detect candidate hypertable.
hypertable.ChunkPermissions:
    Detect bad chunk permissions.
compression.LinearSegmentBy:
    Detect segmentby column for compressed table.
compression.PointlessSegmentBy:
    Detect pointless segmentby column in compressed table.
mats@fury:~/work/timescale/doctor$ timescale-doctor --list=index.* --show=brief
index.UnusedIndex:
    Find all unused indexes.
mats@fury:~/work/timescale/doctor$ timescale-doctor --list=index.*
index.UnusedIndex:
    Find all unused indexes.
```